pkg/scheduler/uthelper/helper.go

```diff
@@ -156,11 +151,19 @@ func (test *TestCommonStruct) CheckBind(caseIndex int) error {
 		}
 	}
 
-	if len(test.Bind) != len(binder.Binds) {
-		return fmt.Errorf("case %d(%s) check bind: \nwant: %v\n got %v ", caseIndex, test.Name, test.Bind, binder.Binds)
+	// in case expected test.BindsNum is 0, but actually there is a binding and wait the binding goroutine to run
+	select {
+	case <-time.After(50 * time.Millisecond):
```

Comment:
Why 50?

Reply:
Just wait a period time and let other goroutines scheduled and run.